steps/bitrise-step-loco/0.0.4/step.yml

```diff
@@ -0,0 +1,362 @@
+    title: Tag updated
+- opts:
+    category: Import
+    summary: Remove existing tags from assets that are MODIFIED during import
+    title: Untag updated
+  untag_updated: null
+- import_tag_absent: null
+  opts:
+    category: Import
+    summary: Tag existing assets in the project that are NOT found in the imported
+      file
+    title: Tag absent
+- import_untag_absent: null
+  opts:
+    category: Import
+    summary: Remove existing tags from assets NOT found in the imported file
+    title: Untag absent
+- export_file_ext: all|csv
+  opts:
+    category: Export
+    description: |
+      Target file type specified as a file extension. Required if you are exporting assets.
+      * The options prepended with `all` export all your locales into a single file.
+      * The options prepended with `archive` export all your locales into a zip file that will be extracted.
+      * The options prepended with `locale` export a single locale into a single file.
+    summary: Target file type specified as a file extension. Required if you are exporting
+      assets.
+    title: File extension
+    value_options:
+    - all|json
+    - all|csv
+    - all|html
+    - all|sql
+    - all|tmx
+    - all|xlf
+    - all|yml
+    - archive|arb
+    - archive|csv
+    - archive|html
+    - archive|ini
+    - archive|json
+    - archive|mo
+    - archive|php
+    - archive|po
+    - archive|pot
+    - archive|plist
+    - archive|bplist
+    - archive|properties
+    - archive|res
+    - archive|resx
+    - archive|sql
+    - archive|strings
+    - archive|stringsdict
+    - archive|tmx
+    - archive|ts
+    - archive|txt
+    - archive|xlf
+    - archive|xliff
+    - archive|xml
+    - archive|yml
+    - locale|arb
+    - locale|csv
+    - locale|html
+    - locale|ini
+    - locale|json
+    - locale|mo
+    - locale|php
+    - locale|po
+    - locale|pot
+    - locale|plist
+    - locale|bplist
+    - locale|properties
+    - locale|res
+    - locale|resx
+    - locale|sql
+    - locale|strings
+    - locale|stringsdict
+    - locale|tmx
+    - locale|ts
+    - locale|txt
+    - locale|xlf
+    - locale|xliff
+    - locale|xml
+    - locale|yml
+- export_locale: null
+  opts:
+    category: Export
+    summary: Locale to export, specified as short code or language tag. e.g. `en`
+      or `en_GB`. Required and only valid if you selected an extension prepended with
+      `locale`.
+    title: Locale
+- export_file_path: null
+  opts:
+    category: Export
+    summary: Where the exported file should be stored. This is required when you are
+      exporting. This should be a path to a file when you are exporting a multi-locale
+      or a single-locale file. This should be a path to a directory when you are exporting
+      an archive.
+    title: File path
+- export_format: default
+  opts:
+    category: Export
+    summary: More specific format of file type. e.g. `rails` applies to `yml` and
```

Comment:
Please clarify what the `default` value means. You may need to put it in a `description` field if it exceeds the limit of the `summary` to be displayed correctly (ca. 100 characters).